src/doc/rustdoc/src/how-to-write-documentation.md

```diff
@@ -267,6 +267,22 @@ you can wrap it like this:
 /// more documentation
 ```
 
+Please note that if you want to put markdown in the HTML tag and for it to
+be interpreted as such, you need to have an empty line between the HTML tags
+and your markdown content. For example if you want to use a link:
+
+```md
```

Comment:
I'm kinda surprised that we use `md` instead of `rs` given the `///` at the start of the line but I get that that probably leads to better highlighting (it's also used that way above).

I've just noticed that we use `text` for a lot of markdown snippets in this file, should probably be changed at some point 🤔